content/nginxaas-azure/changelog.md

```diff
@@ -11,6 +11,12 @@ To see a list of currently active issues, visit the [Known issues]({{< relref "/
 
 To review older entries, visit the [Changelog archive]({{< relref "/nginxaas-azure/changelog-archive" >}}) section.
 
+## March 5, 2025
+
+- {{% icon-info %}} **Retirement of Standard Plan**
+
+   The `Standard` plan for NGINXaaS for Azure has been retired, and users cannot create new deployments with it anymore. If you have a deployment running on the `Standard` plan, consider [migrating]({{< relref "/nginxaas-azure/troubleshooting/migrate-from-standard.md">}}) it to the [`Standard V2 plan`]({{< relref "/nginxaas-azure/billing/overview.md#standard-v2-plan" >}}) without incurring downtime, and to access new features such as NGINX App Protect WAF and additional listen ports.
```

Comment:
```suggestion
   The `Standard` plan for NGINXaaS for Azure has been retired, and you can no longer use it to create new deployments. If you have a deployment running on the `Standard` plan, consider [migrating]({{< relref "/nginxaas-azure/troubleshooting/migrate-from-standard.md">}}) it to the [`Standard V2 plan`]({{< relref "/nginxaas-azure/billing/overview.md#standard-v2-plan" >}}) to access new features such as NGINX App Protect WAF and additional listen ports. Plan migration requires no downtime.
```

Reply:
You know, when I was writing this up, these changes you suggested were what I had in mind and then I just chose mine. 😆 

I will update it 👍

Reply:
reworded it 👍